packages/core/src/conversation/ConversationService/ConversationService.types.ts

```diff
@@ -100,11 +100,11 @@ export type SendMlsMessageParams = SendCommonParams & {
   protocol: ConversationProtocol.MLS;
 };
 
-export type QualifiedUsers = QualifiedId & {skipOwn?: string};
+export type KeyPackageClaimUser = QualifiedId & {skipOwnClientId?: string};
```

Comment:
QualifiedUser seems to be too generic name, with new name we know what this type is being used for.
`skipOwn` -> `skipOwnClientId` - it was hard to say what type of value we expect here, it did sound more like a boolean value.